tests/tensor_ros_src/runTest.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+##
+## @file runTest.sh
+## @author Wook Song <wook16.song@gmail.com>
+## @date  2월 28 2019
+## @brief SSAT test cases for NNStreamer-ros
+#
+if [[ "$SSATAPILOADED" != "1" ]]
+then
+	SILENT=0
+	INDEPENDENT=1
+	search="ssat-api.sh"
+	source $search
+	printf "${Blue}Independent Mode${NC}
+"
```

Comment:
Do you want to add **"** at the new line?

Reply:
Nope. Thanks, I will fix it.